build-scripts/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+# #### Build AWS SDK Client for S3
+
+# Doc link: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/building-sdk-for-browsers.html#using-command-line-tools
+
+# Clone aws-sdk repo
+git clone git@github.com:tmathern/aws-sdk-js.git
+git checkout build-for-node
+cd aws-sdk-js
+
+# Choose version
+# TODO: turn into env variable
+git checkout v.2.686.0
```

Comment:
It would be great to have the `aws-sdk` version encoded in the `package.json` somehow, for visibility and tooling (e.g. dependabot, snyk etc.).

I think `devDependencies` might work, as this is not forced upon downstream modules, but it will install locally, and we need it for the build (and publishing) of this module.

IIUC, the necessary build scripts are not part of the published `aws-sdk` module, but only in their git repo?
- if part (I guess unlikely, but still): then could just cd into `node_modules/aws-sdk` and run from there
- if not part: parse the version from `devDependencies` using e.g. `node -p "require('aws-sdk/package.json').version"`

Reply:
Just checked - they are part of the published module. So we'll be able to leverage this if/once the changes get merged. (I'll update the script then)